src/components/CookieRejectModal.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash/fp'
+import { h } from 'react-hyperscript-helpers'
+import Modal from 'src/components/Modal'
+import { useStore } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+
+
+const CookieRejectModal = () => {
+  const { cookiesAccepted } = useStore(authStore)
+  return cookiesAccepted === false && h(Modal, {
+    title: 'Cookies are required for Terra',
+    showCancel: false,
+    showX: false,
+    onDismiss: () => authStore.update(_.set('cookiesAccepted', true))
```

Comment:
My one problem with this is that clicking off the modal seems to also close it, and I kinda want `OK` to be the only clickable option, but I don't know if our modal is structured for that ATM

Reply:
Escape also closes it. I personally feel like there should be an option for someone to say "I really don't want to use Cookies… and I understand that means I can't use Terra".

Reply:
```suggestion
    shouldCloseOnOverlayClick: false,
    shouldCloseOnEsc: false,
    onDismiss: () => authStore.update(_.set('cookiesAccepted', true))
```
We can prevent the modal from doing this by adding to the properties!

Reply:
@cahrens I agree - this is meant as an iterative improvement over what we have right now. If we want the site to be usable without cookies, that is a _behavior_ change over what has been decided for Terra UI, and would require us to design that feature. (which would be a sizable amount of work). I _definitely_ wouldn't want to block this PR on figuring out how to make Terra usable for non-cookie users.

Reply:
It is better now that Escape and clicking outside do not dismiss the dialog.